internal/proxy/impl.go

```diff
@@ -152,6 +153,160 @@ func (node *Proxy) InvalidateCollectionMetaCache(ctx context.Context, request *p
 	}, nil
 }
 
+func (node *Proxy) CreateDatabase(ctx context.Context, request *milvuspb.CreateDatabaseRequest) (*commonpb.Status, error) {
+	if !node.checkHealthy() {
+		return unhealthyStatus(), nil
+	}
+
+	ctx, sp := otel.Tracer(typeutil.ProxyRole).Start(ctx, "Proxy-CreateDatabase")
+	defer sp.End()
+
+	method := "CreateDatabase"
+	tr := timerecord.NewTimeRecorder(method)
+
+	metrics.ProxyFunctionCall.WithLabelValues(strconv.FormatInt(paramtable.GetNodeID(), 10), method, metrics.TotalLabel).Inc()
+
+	cct := &createDatabaseTask{
+		ctx:                   ctx,
+		Condition:             NewTaskCondition(ctx),
+		CreateDatabaseRequest: request,
+		rootCoord:             node.rootCoord,
+	}
+
+	log := log.With(zap.String("traceID", sp.SpanContext().TraceID().String()),
+		zap.String("role", typeutil.ProxyRole),
+		zap.String("dbName", request.DbName))
+
+	log.Info(rpcReceived(method))
```

Comment:
should be log.Debug like other method in this file.

Reply:
Is it should be Info level for all DDL operations?